python/ray/state.py

```diff
@@ -288,6 +290,9 @@ def object_table(self, object_id=None):
             # Return information about a single object ID.
             return self._object_table(object_id)
         else:
+            # Find the local driver / job
+            global_worker = ray.WorkerID(ray.worker.global_worker.worker_id)
```

Comment:
Isn't it supposed to be `global_worker_id`?

Reply:
hmm. I am not sure I found that . are you referring to `ray.worker.global_worker_id` ?

Reply:
oh what I meant was the variable name. 
```suggestion
            global_worker_id = ray.WorkerID(ray.worker.global_worker.worker_id)
```